docs/usage/configuration-options.md

```diff
@@ -2135,6 +2135,38 @@ For example to apply a special label for Major updates:
 }
 ```
 
+### matchConfidence
+
+<!-- prettier-ignore -->
+!!! warning
+This configuration option is dependent on a Mend API key, and is in private beta testing only. API keys are not available for free or via this repository.
```

Comment:
```suggestion
!!! warning
    This configuration option needs a Mend API key, and is in private beta testing only.
    API keys are not available for free or via the `renovatebot/renovate` repository.
```

The formatting was nearly right, you need 4 spaces on each line. 😉 While I'm at it, a small rewrite to clarify.